### PR TITLE
Remove Travis check from mergify since lint is done in Github action

### PR DESCRIPTION
**Pull request by thom-at-redhat**
# Description

Comment out Travis from Mergify rules
Fixed typo where PR check rerun didn't require 2 approved-reviews

Fixes # (issue)

## Type of change

What is it?

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [ ] Tests update
- [X] Refactor

# Checklist:

- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [X] I run `make pre-commit` to check fmt/vet/lint/test-no-fdo

### CODE REVIEW

**Review by acosferreira — APPROVED** [review]
lgtm

File: .github/mergify.yml [hunk 1]
```diff
@@ -32,14 +32,14 @@ pull_request_rules:
       - and:
           - base=main
           - "#changes-requested-reviews-by=0"
-          - "#approved-reviews-by>=1"
+          - "#approved-reviews-by>=2"
```
Comment (de1987) [inline .github/mergify.yml:35]: nice catch

**Review by de1987 — APPROVED** [review]

File: .github/mergify.yml (hunk 1 above)
Comment (AaronH88) [reply .github/mergify.yml:35]: If I understand this condition correctly, this is the ability to run `retest` . I left it at one as I think anyone should be able to request a `retest` and not need a second person to confirm

**Review by AaronH88 — CHANGES_REQUESTED** [review]

File: .github/mergify.yml [hunk 2]
```diff
@@ -19,7 +19,7 @@ pull_request_rules:
       - author~=^dependabot(|-preview)\[bot\]$
       - "#approved-reviews-by>=2"
       - check-success=Lint
-      - check-success=Travis CI - Pull Request
+      # - check-success=Travis CI - Pull Request
```
Comment (AaronH88) [inline .github/mergify.yml:22]: Feel free to delete these, not just comment it out. It is highly unlikely travis will be coming back

Comment (thom-at-redhat) [reply .github/mergify.yml:22]: as we later discussed, will be done in a upcoming patch :-)

**Pull request merged**